test/e2e/framework/pod/resize.go

```diff
@@ -167,6 +170,7 @@ func makeResizableContainer(tcInfo ResizableContainerInfo) v1.Container {
 func MakePodWithResizableContainers(ns, name, timeStamp string, tcInfo []ResizableContainerInfo) *v1.Pod {
 	testInitContainers, testContainers := separateContainers(tcInfo)
 
+	minGracePeriodSeconds := kuberuntime.MinimumGracePeriodInSeconds
```

Comment:
nit: the comment on this field says

> The value zero indicates stop immediately via ...

So I think you can just set it to zero here, and avoid pluming the `MinimumGracePeriodInSeconds` constant through?